var/spack/repos/builtin/packages/pfunit/package.py

```diff
@@ -55,7 +55,6 @@ class Pfunit(CMakePackage):
 
     variant(
         "max_array_rank",
-        values=int,
```

Comment:
Without adding a set of allowed values in place of this, any string is a valid value for the variant. So I think this PR introduces a new bug. The long string in `spack info` comes from the fact that, if a function is used as a validator, the description of it is its docstring. 

That said, if you can turn this into a variant with a finite set of allowed values, it would be great.

Reply:
@alalazo While @tclune would know better than me, I think the maximum array rank in Fortran is 15, so I'll make a values tuple from that. Please see latest commit.

Reply:
Note also, I have no idea how this scales. I've never done anything other than 5!

Reply:
> Note also, I have no idea how this scales. I've never done anything other than 5!

If it is fine for maintainers of the package to remove the variant completely and use only `5` in the recipe that would be fine with me too. Let me know which option you prefer. Like it is now it's also fine with me.